### PR TITLE
Documentation: Booster infrastructure advantages

### PR DESCRIPTION
**Pull request by AdrianLorenzoDev**
## Description
<Description of the pull request>

This pull request tries to **clarify a little bit better some of the advantages of using Booster regarding the application infrastructure.**

## Changes
- Modified `Why use Booster?` to clarify some of the advantages regarding the deployed application infrastructure.

## Checks
- [X] Project Builds
- [X] Project passes tests and checks
- [X] Updated documentation accordingly

### CODE REVIEW

**Review by davidverdu — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ Events are the cornerstone of a Booster application, and that’s why we say tha
 
 Before Booster, building an event-driven system with the mentioned characteristics required huge investments in hiring engineers with the needed expertise. Booster packs this expertise, acquired from real-case scenarios in high-scale companies, into a very simple tool that handles with the hard parts for you, even provisioning the infrastructure!
 
-We have redesigned the whole developer experience from scratch, taking advantage of the advanced TypeScript type system and Serverless technologies to go from project generation to a production-ready real-time GraphQL API that can ingest thousands of concurrent users in a matter of minutes.
+We have redesigned the whole developer experience from scratch, taking advantage of the advanced TypeScript type system and Serverless technologies to go from project generation to a production-ready application in the cloud composed by a set of orchastrated services which provides a real-time GraphQL API that can ingest thousands of concurrent users in a matter of minutes.
```
Comment (rdoria1) [inline docs/README.md:127]: `application in the cloud composed by a set of orchastrated services which provides` -> `application in the cloud which provides` 

`orhcestrated services` sounds weird, saying we are in the cloud already infers we are using their services.

File: docs/README.md [hunk 2]
```diff
@@ -150,6 +150,7 @@ Booster will fit like a glove in applications that are naturally event-driven li
...
 - **Event-sourcing by default**: Booster keeps all incremental data changes as events, indefinitely. This means that any previous state of the system can be recreated and replayed at any moment, enabling a whole world of possibilities for troubleshooting and auditing, syncing environments or performing tests and simulations.
 - **Booster makes it easy to build enterprise-grade applications**: Implementing an event-sourcing system from scratch is a challenging exercise that usually requires highly specialized experts. There are some technical challenges like eventual consistency, message ordering, and snapshot building. Booster takes care of all of that and more for you, lowering the curve for people that are starting and making expert lives easier.
+- **Choose your application infrastructure and don't get tied to a certain platform:** Booster provides a highly decoupled architecture that enables the possibility of integrating with ease new providers with different specifications, including a custom Multi-cloud provider, without affecting the framework specification.
```
Comment (rdoria1) [inline docs/README.md:153]: `Choose your application infrastructure and don't get tied to a certain platform` -> `Choose your application cloud and avoid vendor locking` This is actually the way it's usually said. The test is fine.
agreed?

Comment (AdrianLorenzoDev) [reply docs/README.md:153]: Awesome @rdoria1! I'm working on that. Thank you :D

File: docs/README.md (hunk 1 above)
Comment (AdrianLorenzoDev) [reply docs/README.md:127]: You are right. I will change that! Thank you :D

**Review by Moneyba — APPROVED** [review]
LGTM

**Pull request merged**